arroyo/types.py

```diff
@@ -67,7 +67,7 @@ def position_to_commit(self) -> Position:
         return Position(self.next_offset, self.timestamp)
 
 
-@dataclass(frozen=True)
+@dataclass(order=True, unsafe_hash=True)
```

Comment:
I think `order` is not strictly necessary?

Reply:
Yeah. I put it in because it matches the rest of the types in this class, and I think that ordering on int/timestamp field can make sense.